docs/basics.md

```diff
@@ -222,6 +222,17 @@ For example:
 - Another possible value for `extractorfunc` is `client.ip` which will categorize requests based on client source ip.
 - Lastly `extractorfunc` can take the value of `request.header.ANY_HEADER` which will categorize requests based on `ANY_HEADER` that you provide.
 
+Sticky sessions are supported with both load balancers. When sticky sessions are enabled, a cookie called `_TRAEFIK_BACKEND` is set on the initial
```

Comment:
indeed it is - I changed the code to `_TRAEFIK_BACKEND` as that's more consistently named with the rest of the code